README.md

```diff
@@ -110,6 +111,9 @@ bundle exec rspec -f d
 - [Twitter](https://twitter.com/mahammad_mostfa)
 - [LinkedIn](https://linkedin.com/in/mahammad-mostafa)
 
+📌 **Zabihullah:**
+- [GitHub](https://github.com/ZabihullahNooriWardak)
+
 <p align="right"><a href="#title">back to top</a></p>
 
 <!-- FUTURE -->
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍